### DISCUSSION TITLE
How to set a fixed height for the project cards?

### DISCUSSION THREAD

**Discussion by clairedavid** [alshedivat/al-folio#2244] [Q&A] [answered]
The rendering in /projects/ is great, but I would prefer the cards to have a fixed height, ideally 400px.
Note: all my cards have images of the same format, the difference in length is due to titles a bit longer than others.

I first tried - naively - to force the grid-item in `_base.scss`:

```
  .grid-sizer,
  .grid-item {
    width: 300px;
    height: 400px;
    margin-bottom: 10px;
  }
```

The new width of 300px looks great. But the height command did not have any effect.

I see that each grid-item element has `position: absolute;` while inspecting in the browser. 
As it is not specified in `_base.scss`, I would like to know where/how it is set up so that I can deactivate it and then add (please comment if it is not the right way): 
```
.grid {
    display: grid;
    grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
}
```

I welcome of course other suggestions if it can work with the absolute positioning. 
For the sake of getting beautiful cards all the same height.
Thanks a lot!

**Comment by george-gca** [comment, marked as answer]
I believe this was fixed in #2357.
